Extension/src/LanguageServer/cppBuildTaskProvider.ts

```diff
@@ -361,6 +361,11 @@ class CustomBuildTaskTerminal implements Pseudoterminal {
         // Do build.
         let command: string = util.resolveVariables(this.command);
         let activeCommand: string = command;
+
+        // Create the exe folder path if it doesn't exists.
```

Comment:
Typo: exist**s** -> exist

Reply:
Thanks -- I've added the fix to https://github.com/microsoft/vscode-cpptools/pull/9270